OpenMRSFormatter.xml

```diff
@@ -81,7 +81,7 @@
         <setting id="org.eclipse.jdt.core.formatter.insert_space_before_opening_angle_bracket_in_type_parameters" value="do not insert"/>
         <setting id="org.eclipse.jdt.core.formatter.insert_new_line_in_empty_type_declaration" value="do not insert"/>
         <setting id="org.eclipse.jdt.core.formatter.comment.clear_blank_lines_in_block_comment" value="true"/>
-        <setting id="org.eclipse.jdt.core.formatter.lineSplit" value="125"/>
+        <setting id="org.eclipse.jdt.core.formatter.lineSplit" value="1250"/>
```

Comment:
Can we ignore such files form being pushed to master? maybe we  include it in a gitignore file?

Reply:
I included it intentionally with [this plugin](https://github.com/esaude/openmrs-module-eptsreports/blob/master/pom.xml#L95) so that the code can properly be formatted during maven build.

Reply:
I think we can leave this here. It is good that we use this to maintain a consistent formatting across the module.

Only possible improvement is maybe to reference the remote location of this config file via a url instead of storing it locally. But this config has not been updated in a few years, so not critical at the moment.